src/components/Patient/TreatmentSummary.tsx

```diff
@@ -479,7 +479,7 @@ export default function TreatmentSummary({
                         ? new Date(
                             diagnosis.onset.onset_datetime,
                           ).toLocaleDateString()
-                        : undefined,
+                        : "- ",
```

Comment:
the undefined case is already being handled in Print Table component checkout line 59 in PrintTable


```suggestion
                        : undefined,
````